src/QueueableAction.php

```diff
@@ -2,10 +2,13 @@
 
 namespace Spatie\QueueableAction;
 
+use Illuminate\Contracts\Queue\Job;
 use Spatie\QueueableAction\Exceptions\InvalidConfiguration;
 
 trait QueueableAction
 {
+    public ?Job $job;
```

Comment:
Related to this:
https://github.com/spatie/laravel-queueable-action/blob/11db345cf5343656e0fd6148198d65b681555a4f/src/ActionJob.php#L102-L107

Assigning dynamic properties causes errors on PHP >= 8.3.